src/google-maps/map-info-window/map-info-window.scss

```diff
@@ -0,0 +1,5 @@
+.map-info-window-container {
+  .map-info-window-content {
+    display: none;
```

Comment:
Add a comment that explains why this is always `display: none`? I don't quite follow it myself

Reply:
Done.